searchlib/src/vespa/searchlib/attribute/address_space_components.h

```diff
@@ -0,0 +1,21 @@
+// Copyright Verizon Media. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root.
+
+#pragma once
+
+#include <vespa/vespalib/stllike/string.h>
+#include <vespa/vespalib/util/address_space.h>
+
+namespace search {
+
+/**
+ * A set of components in an attribute vector that use address space.
+ */
+class AddressSpaceComponents {
+public:
+    static vespalib::AddressSpace default_enum_store_usage();
+    static vespalib::AddressSpace default_multi_value_usage();
+    static vespalib::string enum_store;
+    static vespalib::string multi_value;
```

Comment:
Should these string instances be explicitly marked `const` to prevent any accidental meddling with data that presumably should be considered immutable?

Reply:
Fixed